README.md

```diff
@@ -1,2 +1,34 @@
 Ethereum On-Boarding Testsuite
-==============================
\ No newline at end of file
+==============================
+
+## Implement a Single Node Ethereum Test Network
+
+1. Create an account on https://kurtosistech.com if you don't have one yet.
+2. Verify that the Docker daemon is running on your local machine.
+3. Clone this repository by running `git clone https://github.com/kurtosis-tech/kurtosis-onboarding-experience.git`
+4. Run the empty Ethereum single node test `my_test` to verify that everything works on your local machine.
+   1. Run `bash scripts/build-and-run.sh all`
+   2. Verify that the output of the build-and-run.sh script indicates that one test ran (my_test) and that it passed.
+5. Import the dependencies that are used in this example test suite.
+   1. Run `go get github.com/ethereum/go-ethereum`
+   2. Run `go get github.com/palantir/stacktrace`
+   3. Run `go get github.com/sirupsen/logrus`
+6. Write the Setup() method on the Ethereum single node test in order to bootstrap the network and leave it running and ready to use it.
+   1. In your preferred IDE, open the Ethereum single node test `my_test` at `testsuite/testsuite_impl/my_test/my_test.go`
+   2. Implement the private helper function `getEthereumServiceConfigurations` used to set the Ethereum node service inside the testsuite
+      1. Add the following private helper functions `getContainerCreationConfig()`, `getRunConfigFunc()` and `getEthereumServiceConfigurations()` in [this Gist](https://gist.github.com/leoporoli/123cb1d682d74dcafe7920f01809b167) to the bottom of the file, so they can be used later. 
```

Comment:
I really like the impetus of using gists, though I'd actually prefer if we inline the code here. Rationale:
1) Then the code that the user copies gets version-controlled here
2) Less clicks for the user

You can add code blocks to a checklist item by indenting an extra level like so:

1. Item1
2. Item2
    1. Item 2.1
        ```
        foobar := "bang"
        blork := foobar + "bor"
        ```